sdk/ml/azure-ai-ml/tests/test_configs/workspace_connection/custom.yaml

```diff
@@ -0,0 +1,8 @@
+name: test_ws_conn_custom_keys
+type: custom_keys
+target: my_endpoint
+tags:
+  one: two
+credentials:
+  type: api_key
```

Comment:
Is this supposed to be custom_key ?

Reply:
No, that's the credential type, which for custom keys is (afaik) flexible. The custom_keys is the overall type located further up in that file.